src/components/banner/index.tsx

```diff
@@ -82,71 +82,100 @@ export const Banner = (props: IBannerProps) => {
     cursor: 'pointer'
   };
   const defaultTextParentStyles = {
-    flex: '1'
+    flex: '1',
+    maxWidth: 'calc(100% - 80px)'
   };
 
   return (
-    <div
-      className="banner"
-      style={{
-        ...defaultBannerStyles,
-        ...BannerStyle
-      }}
-      onClick={onClickView}
-    >
+    <>
+      <style>
+        {`
+        @media screen and (max-width: 800px) {
+            .titleText {
+              overflow: hidden;
+              text-overflow: ellipsis;
+              max-height: 2.6em;
+              line-height: 1.3em;
+            }
+            .banner {
+              height: 250px;
+              display: flex;
+              flex-direction: column;
+            }
+          }
+        `}
```

Comment:
I'd prefer to keep our styles and our components separate. Please, at a minimum, pull this out into its own constant.

Reply:
Done